avaframe/runDamBreak.py

```diff
@@ -47,7 +47,7 @@
 Particles, Fields, Tsave, dem, plotDict, reportDictList = runCom1DFA.runCom1DFA(avaDir=avaDir, cfgFile=damBreakCfg)
 
 # create simDict of results
-inputDir = 'data/avaDamBreak/Outputs/com1DFAPy/peakFiles/timeSteps'
+inputDir = 'data/avaDamBreak/Outputs/com1DFA/peakFiles/timeSteps'
```

Comment:
This fails on windows. I suggest we start using pathlib for stuff like this (as long as it is in a run script)

Reply:
switched to pathlib